src/mscorlib/src/System/ReadOnlySpan.cs

```diff
@@ -328,7 +328,7 @@ public T[] ToArray()
         public static ReadOnlySpan<T> Empty => default(ReadOnlySpan<T>);
     }
 
-    public static class ReadOnlySpanExtensions
+    public static class ReadOnlySpanExtensionMethods
```

Comment:
@jkotas, does this work? Shall I move all these extension methods over to corefx?